syne_tune/experiments.py

```diff
@@ -61,30 +64,87 @@ def creation_date(self):
         """
         return datetime.fromtimestamp(self.metadata[ST_TUNER_CREATION_TIMESTAMP])
 
-    def plot(self, **plt_kwargs):
+    def plot_hypervolume(
+        self, reference_points: np.ndarray = None, figure_path: str = None, **plt_kwargs
+    ):
+        """Plot best hyervolume value as function of wallclock time
+
+        :param reference_points: Reference points for hypervolume calculations.
+                                 If None, the maximum values of each metric is used.
+        :param figure_path: If specified, defines the path where the figure will be saved.
+                            If None, the figure is shown
+        :param plt_kwargs: Arguments to :func:`matplotlib.pyplot.plot`
+        """
+        import matplotlib.pyplot as plt
+
+        metric_names = self.metric_names()
```

Comment:
You would very likely want to customize the metrics that can be passed here, ie to be able to pass `metric_names` as argument and use the full list by default (as you often want to optimize only a subset of available metrics, for instance in the benchmark in blackbox repository which contains many metrics).